invoice/src/main/java/org/killbill/billing/invoice/InvoicePluginDispatcher.java

```diff
@@ -236,15 +236,19 @@ public SplitInvoiceResult splitInvoices(final DefaultInvoice originalInvoice,
                                               final boolean isDryRun,
                                               final CallContext callContext,
                                               final Iterable<PluginProperty> pluginProperties,
+                         					  final LocalDate targetDate,
+                         					  final List<Invoice> existingInvoices,
+                         					  final boolean isRescheduled,                                              
                                               final InternalTenantContext tenantContext) {
         log.debug("Invoking invoice plugins for splitInvoices operation: isDryRun='{}', originalInvoice='{}'", isDryRun, originalInvoice);
 
         Iterable<PluginProperty> inputPluginProperties = pluginProperties;
         final Collection<InvoicePluginApi> invoicePlugins = getInvoicePlugins(tenantContext).values();
         final Invoice clonedInvoice = (Invoice) originalInvoice.clone();
         for (final InvoicePluginApi invoicePlugin : invoicePlugins) {
-            final InvoiceGroupingResult grpResult = invoicePlugin.getInvoiceGrouping(clonedInvoice, isDryRun, inputPluginProperties, callContext);
-            if (grpResult !=  null) {
+        	final InvoiceContext invoiceContext = new DefaultInvoiceContext(targetDate, clonedInvoice, existingInvoices, isDryRun, isRescheduled, callContext);
```

Comment:
Nit: indent

Reply:
Did not understand this, but I've tried to fix the indentation issue. Please let me know if you still find an issue.